deployment/helm/synopsys-alert/values.yaml

```diff
@@ -47,6 +47,17 @@ cfssl:
 postgres:
   registry: "docker.io/centos" # override the docker registry at container level
   isExternal: false # false for running Postgres as a container and true for using External Postgres database
+  ssl: true # If true, Alert uses SSL for external Postgres connection
```

Comment:
is this supposed to be true by default going forward?

Reply:
I think it would make sense if it were false for Alert. Will change that unless others object